src/functionalTest/java/uk/gov/hmcts/sptribs/citizen/event/CicSubmitCaseEventFT.java

```diff
@@ -15,26 +16,70 @@
 
     private static final String REQUEST = "classpath:request/casedata/ccd-callback-casedata-submit-case-about-to-start.json";
+    private static final String REQUEST_SUBMITTED = "classpath:request/casedata/ccd-callback-casedata-submit-case-submitted.json";
+    private static final String REQUEST_MISSING_CASE_NUMBER =
+        "classpath:request/casedata/ccd-callback-casedata-submit-case-missing-case-number.json";
+    private static final String REQUEST_MISSING_PARTIES =
+        "classpath:request/casedata/ccd-callback-casedata-submit-case-missing-parties.json";
+
     private static final String RESPONSE = "classpath:responses/response-caseworker-submit-case-about-to-submit.json";
+    private static final String RESPONSE_NOTIFY_SUCCESS = "classpath:responses/response-cic-submit-case-notification-success.json";
+    private static final String RESPONSE_NO_NOTIFY = "classpath:responses/response-cic-submit-case-no-notification.json";
+    private static final String RESPONSE_NOTIFY_FAILED = "classpath:responses/response-cic-submit-case-notification-failed.json";
 
     private static final String CITIZEN_SUBMIT_CASE_EVENT_ID = "citizen-cic-submit-dss-application";
 
     @Test
     public void shouldSuccessfullySubmitCaseWhenAboutToSubmitCallbackIsTriggered() throws Exception {
         final Map<String, Object> caseData = caseData(REQUEST);
-
         final Response response = triggerCallback(caseData, CITIZEN_SUBMIT_CASE_EVENT_ID, ABOUT_TO_SUBMIT_URL);
-        assertThat(response.getStatusCode()).isEqualTo(OK.value());
 
+        assertThat(response.getStatusCode()).isEqualTo(OK.value());
         assertThatJson(response.asString())
             .when(IGNORING_EXTRA_FIELDS)
             .when(IGNORING_ARRAY_ORDER)
             .isEqualTo(json(expectedResponse(RESPONSE)));
     }
+
+    @Disabled // Should be enabled after removing the temporary toggle DSS_FRONTEND_SUBMIT_NOTIFICATION_ENABLED
+    @Test
+    public void shouldReceiveNotificationWhenSubmittedCallbackIsInvoked() throws Exception {
+        final Map<String, Object> caseData = caseData(REQUEST_SUBMITTED);
+        final Response response = triggerCallback(caseData, CITIZEN_SUBMIT_CASE_EVENT_ID, SUBMITTED_URL);
+
+        assertThat(response.getStatusCode()).isEqualTo(OK.value());
+        assertThatJson(response.asString())
+            .when(IGNORING_EXTRA_FIELDS)
+            .isEqualTo(json(expectedResponse(RESPONSE_NOTIFY_SUCCESS)));
+    }
+
+    @Test
+    public void shouldNotSendApplicationReceivedNotificationWhenNotifyPartiesNotFound() throws Exception {
+        final Map<String, Object> caseData = caseData(REQUEST_MISSING_PARTIES);
+        final Response response = triggerCallback(caseData, CITIZEN_SUBMIT_CASE_EVENT_ID, SUBMITTED_URL);
+
+        assertThat(response.getStatusCode()).isEqualTo(OK.value());
+        assertThatJson(response.asString())
+            .when(IGNORING_EXTRA_FIELDS)
+            .isEqualTo(json(expectedResponse(RESPONSE_NO_NOTIFY)));
```

Comment:
You can do something like this rather than need a whole separate json file:
        ```
assertThatJson(response.asString())
            .inPath("$.confirmation_header")
            .isEqualTo("# Application Received notification failed %n## Please resend the notification");
```

Reply:
Hmm.. I wanted to maintain as a separate file if we want to assert few other fields in the future. I am ok to remove if you don't see any improvement needed in the assertion at a later stage.

Reply:
As it is a submitted callback we won't expect much else in response